src/driftpy/drift_client.py

```diff
@@ -139,10 +141,24 @@ def __init__(
             StandardTxSender(self.connection, opts) if tx_sender is None else tx_sender
         )
 
+        self.subscribed = False
+        self.inspect_mode = inspect_mode
+
     async def subscribe(self):
+        if not (
```

Comment:
feel like this is a lot of overhead for everybody to need to do this call when subscribing